atom/browser/api/atom_api_web_contents.cc

```diff
@@ -437,8 +437,10 @@ void WebContents::PluginCrashed(const base::FilePath& plugin_path,
 
 void WebContents::DocumentLoadedInFrame(
     content::RenderFrameHost* render_frame_host) {
-  if (!render_frame_host->GetParent())
+  if (!render_frame_host->GetParent()) {
     Emit("dom-ready");
+    managed_web_contents()->SetInspectedWebContents();
```

Comment:
Not every WebContents is managed by InspectableWebContents, this should be moved to NativeWindow.